packages/core-typings/src/IRoom.ts

```diff
@@ -171,7 +171,10 @@ export interface IOmnichannelGenericRoom extends Omit<IRoom, 'default' | 'featur
 	metrics?: any;
 	waitingResponse: any;
 	responseBy: any;
-	priorityId: any;
+	priorityId?: string;
```

Comment:
Wondering if we should remove this field now since after the above migration runs, the workspace won't be having this field